### PR TITLE
Merge privacy-grade

### PR DESCRIPTION
**Pull request by sammacbeth**
**Reviewer:**

## Description:

- Merges the privacy-grade repo into `/packages` as an npm workspace.
- Adds a workflow to run lint and tests for privacy grade.

### CODE REVIEW

File: packages/privacy-grade/src/classes/trackers.js [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+            'about:',
+            'moz-extension:',
+
+            // Special web protocols
+            'file:',
+            'javascript:',
+            'data:',
+            'blob:',
+            'view-source:',
+            'vbscript:',
```
Comment (jonathanKingston) [inline packages/privacy-grade/src/classes/trackers.js:261]: Is this even supported in any browsers?

Comment (sammacbeth) [reply packages/privacy-grade/src/classes/trackers.js:261]: This was triggered by our code scanning. No idea what the attack vector would be here.
Screenshot 2023-03-08 at 13 41 30

Comment (jonathanKingston) [reply packages/privacy-grade/src/classes/trackers.js:261]: Ah, I mean we can leave it in but yeah this iiuc isn't relevant to our use-case at all.

**Review by jonathanKingston — APPROVED** [review]
LGTM, pull the trigger and see if it explodes :D

**Pull request merged**